onnxruntime/core/framework/session_options.h

```diff
@@ -82,7 +82,10 @@ struct SessionOptions {
   // Deterministic compute is likely not as performant. This option is default to false.
   bool use_deterministic_compute = false;
 
-  // Control the pre-packing of initialized constant tensors
-  bool use_prepacking = true;
+  // Stores the configurations for this session
+  std::unordered_map<std::string, std::string> session_configurations;
 };
```

Comment:
Add comment somewhere linking to the header file where all keys should be defined so developers are aware they should be added there.